Gems/ROS2/Code/Include/ROS2/Manipulator/MotorizedJointComponent.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "MotorizedJointBus.h"
+#include "ROS2/VehicleDynamics/DriveModels/PidConfiguration.h"
```

Comment:
This include should use an angle bracket: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#including-headers
```suggestion
#include <ROS2/VehicleDynamics/DriveModels/PidConfiguration.h>
```

Reply:
Done